apps/potlock/widget/Pots/Header.jsx

```diff
@@ -65,14 +65,17 @@ const applicationExists = existingApplication || applicationSuccess;
 
 const now = Date.now();
 const publicRoundOpen = now >= public_round_start_ms && now < public_round_end_ms;
+const publicRoundEnded = now > public_round_end_ms;
 
 const applicationOpen = now >= application_start_ms && now < application_end_ms;
 
 const canApply = applicationOpen && !applicationExists && !userIsChefOrGreater;
 
-const canPayoutsBeSet = userIsChefOrGreater && !cooldown_end_ms && !all_paid_out;
+const canPayoutsBeSet =
+  userIsChefOrGreater && !cooldown_end_ms && !all_paid_out && publicRoundEnded;
 
-const canPayoutsBeProcessed = userIsAdminOrGreater && now >= cooldown_end_ms && !all_paid_out;
+const canPayoutsBeProcessed =
+  userIsAdminOrGreater && now >= cooldown_end_ms && !all_paid_out && publicRoundEnded;
```

Comment:
FYI publicRoundEnded is redundant here, as there can be no cooldown_end_ms if the public round hasn't ended. but no problem to leave it.